### ISSUE TITLE
Regarding the reproduction of the results

### ISSUE THREAD

**Issue by ayclove** [mjmjeong/InfoNeRF#2]
Thanks for your great work.

Regarding reproduction of the lego dataset , training does not converge if I use both entropy and KL loss.
I used the default config file, which is at "config/infonerf/synthetic/lego.txt".
Although I expected to have similar results as mentioned in the paper, but I failed to train the model.
In order to turn on KL loss, I just added "Smoothing=True" and used default parameters for the rest.

----------------------------------------------------------------------------------------------------------------------------
    parser.add_argument("--smooth_sampling_method", type=str, default='near_pose', 
        help='how to sample the near rays, near_pose: modifying camera pose, near_pixel: sample near pixel', 
                    choices=['near_pose', 'near_pixel'])
    # 1) sampling by rotating camera pose
    parser.add_argument("--near_c2w_type", type=str, default='rot_from_origin', 
                        help='random augmentation method')
    parser.add_argument("--near_c2w_rot", type=float, default=5, 
                        help='random augmentation rotate: degree')
    parser.add_argument("--near_c2w_trans", type=float, default=0.1, 
                        help='random augmentation translation')
----------------------------------------------------------------------------------------------------------------------------
Could you please guide me if there are some important hyper-parameters missing in the default config file?
Just for your information, the default config file(w/o KL loss), I succeeded in training and results seem to be reasonable.

In addition, you mentioned the role of KL loss in the appendix.
I would like to reproduce the result with narrow-baseline dataset as well.
Would you please tell me which indexes of images you used for narrow-baseline 4-views?

Thank you for your feedback in advance.

Best regards,

**Comment by mjmjeong** [comment]
For the synthetic dataset, the information gain reduction loss is not necessary because the sampled images usually have wide-baseline. 

For the role of KL loss in the appendix, we used lambda_1 = 0.0005, lambda_2=0.00001. Note that, we sample the near rays from the neighboring pixels for this experiment. 

Index for the narraw baseline: [13, 72, 95, 4]
Index for the wide baseline: [8, 72, 37, 41]

```
# config file
smooth_sampling_method='near_pixel'
smooth_pixel_range=1
smoothing_activation=softmax
smoothing_lambda=0.00001
smoothing_step = 2500
```

**Comment by ayclove** [comment]
Thanks for the comment :)
I have few more questions regarding the experiments.

You mentioned that the results in table 2 are obtained by running experiments 5 times with different viewpoint samples. Would you tell me how you selected the 5 different viewpoint sets in table 2? If possible, I would like to know the way to select scenes for LLFF and DTU dataset as well.
In addition, I am wondering how you calculated the average PSNR and their standard deviations. I tried to come up with the same number in the paper, I failed to get the same standard deviations.

Thank you in advance.
